rotkehlchen/db/dbhandler.py

```diff
@@ -2957,7 +2957,7 @@ def get_bitcoin_xpub_data(
             tags = deserialize_tags_from_db(entry[4])
             result.append(XpubData(
                 xpub=HDKey.from_xpub(entry[0], path='m'),
-                blockchain=SupportedBlockchain.deserialize(entry[1]),  # type: ignore
+                blockchain=SupportedBlockchain(entry[1].upper()),  # type: ignore
```

Comment:
This should change but requires a refactor in the current database upgrade (another PR)